lib/token-verifier/WP_Auth0_SignatureVerifier.php

```diff
@@ -0,0 +1,87 @@
+<?php
```

Comment:
[See this diff](https://github.com/auth0/auth0-PHP/pull/401/files#diff-82a6ce33086c518324a561bdd71b8a9b) for changes from approved PHP SDK ones.